src/library_wasm_worker.js

```diff
@@ -98,8 +99,8 @@ addToLibrary({
     // Run the C side Worker initialization for stack and TLS.
     _emscripten_wasm_worker_initialize(m['sb'], m['sz']);
 #if PTHREADS
-    // Record that this Wasm Worker supports synchronous blocking in emscripten_futex_wake().
-    ___set_thread_state(/*thread_ptr=*/0, /*is_main_thread=*/0, /*is_runtime_thread=*/0, /*supports_wait=*/0);
+    // Record that this Wasm Worker supports synchronous blocking in emscripten_futex_wait().
```

Comment:
I think it probably cleaner to just do this here:

```
#if AUDIO_WORKLET
int supportsFutex = typeof AudioWorkletGlobalScope === 'undefined'";
#else
int supportsFutex = 1;
#endif
```

Rather than defining something that is (a) pulluting the global scope, (b) only used once and (c) far away from the call site

Reply:
Noo, I don't agree. This is cleaner because it won't introduce new `#if`s in the code, and better since it won't require new variables that Closure would need to then clean up.